bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoHandler.java

```diff
@@ -192,7 +195,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             if (command instanceof OnOffType) {
                 try {
                     boolean binaryState = OnOffType.ON.equals(command) ? true : false;
-                    String soapHeader = "\"urn:Belkin:service:basicevent:1#SetBinaryState\"";
+                    String soapHeader = "\"urn:Belkin:service:BASICACTION:1#SetBinaryState\"";
```

Comment:
Is this change intended?

Reply:
No, search/replace error, will fix later…

Reply:
Done